subjects/podiumposition/README.md

```diff
@@ -0,0 +1,38 @@
+## podiumposition
+
+### Instructions
+
+A F1 race just finished and the commentator is calling the finishing positions incorrectly.
+Help to fix this before the contestants arrive at the podium by providing the commentator with the correct podium position.
+
+Write a function `PodiumPosition` that accepts an array of arrays as a `string` and returns the competitor positions correctly.
```

Comment:
Write a function `PodiumPosition` that takes an array of arrays of type `string` and returns ...